.github/workflows/build-kata-static-tarball-amd64.yaml

```diff
@@ -364,3 +358,116 @@ jobs:
           path: kata-static.tar.zst
           retention-days: 15
           if-no-files-found: error
+
+  build-tools-asset:
+    name: build-tools-asset
+    runs-on: ubuntu-22.04
+    permissions:
+      contents: read
+      packages: write
+      id-token: write
+      attestations: write
+    strategy:
+      matrix:
+        asset:
+          - agent-ctl
+          - csi-kata-directvolume
+          - genpolicy
+          - kata-ctl
+          - kata-manager
+          - trace-forwarder
+        stage:
+          - ${{ inputs.stage }}
+    env:
+      PERFORM_ATTESTATION: ${{ matrix.asset == 'agent' && inputs.push-to-registry == 'yes' && 'yes' || 'no' }}
```

Comment:
Should this be dropped or should we make sure we do the attestation for the tools?

In any case this condition is no really correct for the tools.